R/class-workbook.R

```diff
@@ -3791,9 +3792,26 @@ wbWorkbook <- R6::R6Class(
         stop("hidden argument is longer than cols.")
       }
 
+      compatible_length <- length(cols) %% length(widths) == 0
+
+      if (!compatible_length) {
+        # needed because rep(c(1, 2 ), length.out = 3) is successful,
+        # but not clear if this is what the user wanted.
+        stop("`cols` and `widths` don't have compatible lengths.\n",
```

Comment:
Maybe throw a warning for a while? Before we error out on potentially insignificant errors.